docs/api-docs/getting-started/deprecations-and-sunsets.mdx

```diff
@@ -37,6 +37,7 @@ We no longer support the following general product tools.
 |Deprecated tool|Support end date|
 |:--|:--|
 |[Google AMP](https://amp.dev/)|January 18th 2023|
+|[Google's Universal Analytics](https://support.google.com/analytics/answer/11583528?hl=en)|July 1st 2023|
```

Comment:
@bc-sanjayjoshi Was July 1st the date when BC stopped supporting Universal Analytics?

Reply:
Resolving conversation (see 
![UA](https://github.com/bigcommerce/dev-docs/assets/96258747/13c49dda-6af8-4ae3-90c8-97de22d5660f)
screenshot)